src/routes/safe/components/Transactions/TxList/TxSummary.tsx

```diff
@@ -33,54 +89,28 @@ export const TxSummary = ({ txDetails }: Props): ReactElement => {
           <TxShareButton id={txDetails.txId} />
         </div>
       )}
-      <div className="tx-hash">
-        <Text size="xl" strong as="span">
-          Transaction hash:{' '}
-        </Text>
-        {txHash ? (
-          <InlineEthHashInfo textSize="xl" hash={txHash} shortenHash={8} showCopyBtn explorerUrl={explorerUrl} />
-        ) : (
-          <Text size="xl" as="span">
-            {NOT_AVAILABLE}
-          </Text>
-        )}
-      </div>
-      {safeTxHash !== undefined && (
-        <div className="tx-hash">
-          <Text size="xl" strong as="span">
-            SafeTxHash:{' '}
-          </Text>
-          <InlineEthHashInfo textSize="xl" hash={safeTxHash} shortenHash={8} showCopyBtn />
-        </div>
-      )}
-      {nonce !== undefined && (
-        <div className="tx-nonce">
-          <Text size="xl" strong as="span">
-            Nonce:{' '}
-          </Text>
-          <Text size="xl" as="span">
-            {nonce}
-          </Text>
-        </div>
+      {txHash && <TxDataRow title="Transaction hash:" value={txHash} inlineType="hash" />}
+      {safeTxHash && <TxDataRow title="SafeTxHash:" value={safeTxHash} inlineType="hash" />}
+      {created && <TxDataRow title="Created:" value={formatDateTime(created)} />}
+      <TxDataRow title="Executed:" value={executedAt ? formatDateTime(executedAt) : NOT_AVAILABLE} />
+      <br></br>
+
+      {/* Advanced TxData */}
+      {txData?.operation !== undefined && (
+        <TxDataRow title="Operation:" value={`${txData?.operation} (${Operation[txData?.operation].toLowerCase()})`} />
       )}
-      {created && (
-        <div className="tx-created">
-          <Text size="xl" strong as="span">
-            Created:{' '}
-          </Text>
-          <Text size="xl" as="span">
-            {formatDateTime(created)}
-          </Text>
-        </div>
+      {safeTxGas && <TxDataRow title="safeTxGas:" value={safeTxGas} />}
+      {baseGas && <TxDataRow title="baseGas:" value={baseGas} />}
+      {gasPrice && <TxDataRow title="gasPrice:" value={gasPrice} />}
+      {gasToken && <TxDataRow title="gasToken:" value={gasToken} inlineType="hash" />}
+      {refundReceiver && <TxDataRow title="refundReceiver:" value={refundReceiver} inlineType="hash" />}
+      {confirmations?.map(({ signature }, index) => (
+        <TxDataRow title={`Signature ${index + 1} :`} value={signature} inlineType="rawData" key={index} />
```

Comment:
Don't use indexes as keys.

Reply:
There shouldn't be a space before a colon. (The same goes for the following line.)